multiqc/modules/skewer/skewer.py

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env python
+
+""" MultiQC module to parse output from QualiMap """
+
+from __future__ import print_function
+
+import os
+from collections import OrderedDict
+import logging
+import re
+from multiqc import config, BaseMultiqcModule
+
+# Initialise the logger
+log = logging.getLogger(__name__)
+
+
+class MultiqcModule(BaseMultiqcModule):
+    """ Skewer """
+
+    def __init__(self):
+        # Initialise the parent object
+        super(MultiqcModule, self).__init__(name='Skewer', anchor='skewer',
+                                            href="https://github.com/relipmoc/skewer.git",
+                                            info="is a tools to trim adapters off of reads")
```

Comment:
I usually take some form of the description used on the author's web page for this. So would be something like:

> is an adapter trimming tool specially designed for processing next-generation sequencing (NGS) paired-end sequences.